src/main/blackbox/blackbox_fielddefs.h

```diff
@@ -33,6 +33,7 @@ typedef enum FlightLogFieldCondition {
 
     FLIGHT_LOG_FIELD_CONDITION_MAG,
     FLIGHT_LOG_FIELD_CONDITION_BARO,
+	FLIGHT_LOG_FIELD_CONDITION_PITOT,
```

Comment:
Should this not be added onto the end of the `enum` list, so as not to change other values?